__tests__/nc-games.test.js

```diff
@@ -460,4 +460,34 @@ describe("/api/users", () => {
             })
         })
     })
+})
+
+describe("/api/comments/:comment_id", () => {
+    describe("DELETE", () => {
+        test("Status 204: When a valid comment_id is passed with the delete HTTP method, delete the comment from the database and return a 204 status code", () => {
+            return request(app)
```

Comment:
I'd quite like an extra test/ expect statement to check whether the the comment has actually been deleted from the db. For example, you could try to delete the same comment again, but expect a 400 (as you've set up below) to prove that it's not found the second time around

Reply:
Thanks Rose, have added a then() block to run the delete method again and expect '404 comment not found' to confirm it's been deleted!